src/components/AddItem/AddItem.meta.tsx

```diff
@@ -0,0 +1,16 @@
+import { AddItem, AddItemProps } from '.';
+import Registry from '@ui-autotools/registry';
+
+const AddItemMetadata = Registry.getComponentMetadata(AddItem);
+AddItemMetadata.nonReactStrictModeCompliant = true;
+
+AddItemMetadata.addSim({
+  title: 'render',
+  props: {
+    disabled: false,
+    hasError: false,
+    alignment: 'center',
+    iconSize: 'medium',
+    children: 'Add Item',
+  } as AddItemProps,
+});
```

Comment:
Let's add a test for `disabled: true`

Reply:
Also since the text is optional, and the button can only have an icon, we should add a test with icon only.
But be aware, it will fail 😁